superset-frontend/src/pages/ThemeList/index.tsx

```diff
@@ -215,6 +233,96 @@ function ThemesList({
     addSuccessToast(t('Theme imported'));
   };
 
+  const handleSetSystemDefault = (theme: ThemeObject) => {
```

Comment:
I think for these confirmation handler functions there is some opportunity for optimization / drying up